cypress/integration/types/constants.ts

```diff
@@ -63,6 +63,9 @@ export const tagName = "Tag name";
 export const optionMenu = '[aria-label="Options menu"]';
 export const userPerspectiveMenu = ".pf-c-select__menu-item";
 
+//React Component Names
+export const reactComponentNameSelect = "Select";
```

Comment:
Just as a way to easy found react and non react, to check it they are more stable.
But it has a different log signature.
will remove the prefix

Reply:
Those are meant to keep React **Component Names** 
which is a base for locating by react
so without the ***reactComponentName***  prefix, it would be just ***Select***.